lib/reference/functions.mjs

```diff
@@ -32,6 +32,7 @@ const singleArgumentMathFunctions = new Set([
 	'asin',
 	'atan',
 	'calc',
+	'calc-size',
```

Comment:
[suggestion] According to the [CSS spec](https://drafts.csswg.org/css-values-5/#calc-size), `calc-size` receives two arguments:

```
<calc-size()> = calc-size( <calc-size-basis>, <calc-sum> )
```

So adding `calc-size` to `singleArgumentMathFunctions` is not right. Can you add it to `mathFunctions` instead?

https://github.com/stylelint/stylelint/blob/130c0919f4132606a81447fa70c990e286963b8a/lib/reference/functions.mjs#L44